setup/management.sh

```diff
@@ -61,6 +61,41 @@ if [ ! -f $STORAGE_ROOT/backup/secret_key.txt ]; then
 	$(umask 077; openssl rand -base64 2048 > $STORAGE_ROOT/backup/secret_key.txt)
 fi
 
+
+# Download jQuery and Bootstrap local files
+echo "Installing Mail-in-a-Box system management web assets..."
+
+# jQuery CDN URL
+jquery_version=2.1.4
+jquery_url=https://code.jquery.com
+
+# Bootstrap CDN URL
+bootstrap_version=3.3.7
+bootstrap_url=https://maxcdn.bootstrapcdn.com/bootstrap/$bootstrap_version
+
+js_lib=$STORAGE_ROOT/www/default/assets/js/lib
```

Comment:
This could be simplified by putting all of the assets into a single directory, rather than two.